code/modules/events/swarmer.dm

```diff
@@ -2,7 +2,8 @@
 	name = "Spawn Swarmer Beacon"
 	typepath = /datum/round_event/spawn_swarmer
 	weight = 8
-	max_occurrences = 1 //Only once okay fam
+	//max_occurrences = 1 //Only once okay fam //ORIGINAL
```

Comment:
what's wrong with swarmers?

Reply:
Ever had a round versus them? It's completely atrocious, they keep breaking things, stealing mats, locking people, completely derailing the entire round as they snowball. It can't be a random occurance.

Reply:
I think we need to decide on policy level if we're going to allow the game to run the round for us or the admins, because I'm not sure which will do a better job.

Reply:
I think swarmers should be disabled regardless